packages/backend-modules/assets/express/pdf.js

```diff
@@ -13,15 +13,26 @@ module.exports = (server) => {
       return res.status(403).end()
     }
 
-    // path with query
-    const path = req.originalUrl.replace(/^\/pdf/, '')
+    // get express path
+    const { path } = req.params
 
-    const result = await fetch(`${PDF_BASE_URL}${path}`, {
-      method: 'GET',
-    }).catch((error) => {
-      console.error('pdf fetch failed', { error })
+    // pick query parameters intended for PDF endpoint
+    // (pass along rest as options, later)
+    const { images, download, size, ...options } = req.query
+
+    // build URL to fetch PDF from
+    const fetchUrl = new URL(`${PDF_BASE_URL}/${path}`)
+
+    // add params
+    images && fetchUrl.searchParams.set('images', images)
```

Comment:
Readability says "Big oof"

Reply:
What would Readability make more comfortable? If clauses? map?

(Thought this might be more readable to frontend versed peepz.)